client/js/components/drawer.js

```diff
@@ -0,0 +1,25 @@
+class Drawer {
+  constructor(el) {
+    this.el = el;
+    this.trigger = el.getElementsByClassName('js-trigger')[0];
+    this.trigger.addEventListener('click', this.toggleActive.bind(this));
```

Comment:
The list I came up with covers over 95% of traffic. See below:

Chrome (latest 2 versions)	Full support
Safari (latest 2 versions)	Full support
Firefox (latest 2 versions)	Full support
Edge (latest 2 versions)	        Full support
Internet Explorer (latest 2 versions)	Full support
Android Browser	Full support

Opera Mini (extreme mode)	Basic support 

And providing we're using 'cutting the mustard' and progressive enhancement, anything not supporting addEventListener should get the non javascript version